akka-remote/src/main/scala/akka/remote/AddressUidExtension.scala

```diff
@@ -39,11 +39,14 @@ class AddressUidExtension(val system: ExtendedActorSystem) extends Extension {
     else tlr.nextInt().toLong
   }
 
-  // used by old remoting and part of public api
-  @deprecated("Use longAddressUid instead", "2.4.x")
-  lazy val addressUid: Int = {
+  // private because GenJavaDoc fails on deprecated annotated lazy val
+  private lazy val _addressUid: Int = {
```

Comment:
Can we add a java `@Deprecated` maybe?
Hm the extension seems public though... will trigger MiMa, even if fine to do?